### ISSUE TITLE
Some unit tests failing when run against Chrome

### ISSUE THREAD

**Issue by parlarjb** [rackerlabs/encore-ui#991]
By default, our unit tests are all run against PhantomJS, because it's the only _easy_ way to run them headless.

The exception is when we do `grunt test:debug`. This launches a Chrome browser so we have access to all the debug tools.

We recently found that a few of our unit tests were _only_ failing when run with Chrome (I guess no one had run `grunt test:debug` recently against the full suite). Specifically

```
hrome 43.0.2357 (Mac OS X 10.10.3) rxFormItem textarea special case should put text-area-label class on the main div FAILED
	AssertionError: expected false to be true
Chrome 43.0.2357 (Mac OS X 10.10.3) rxFormUtils getSelectedOptionForTable returns the selected option for the table with id tableId FAILED
	AssertionError: expected undefined to deeply equal { rowId: 'table1_rowId1' }
	    at Context.<anonymous> (base/src/rxForm/rxForm.spec.js?eb992222af5bee3d1e04f8367e8fe4f0534c7b06:445:80)
Chrome 43.0.2357 (Mac OS X 10.10.3) rxFormUtils getSelectedOptionForTabSet returns the selected option for the tabset FAILED
	TypeError: Cannot read property 'rowId' of undefined
	    at Object.rxFormUtils.getSelectedOptionForTabSet (base/src/rxForm/rxForm.js?0d03ed1d97ecc3d5da22e552e530565d150a8130:328:72)
	    at Context.<anonymous> (base/src/rxForm/rxForm.spec.js?eb992222af5bee3d1e04f8367e8fe4f0534c7b06:459:35)
Chrome 43.0.2357 (Mac OS X 10.10.3) rxFormUtils getSelectedOptionForTabSet returns the selected option without the field id if no field-id exists FAILED
	TypeError: Cannot read property 'rowId' of undefined
	    at Object.rxFormUtils.getSelectedOptionForTabSet (base/src/rxForm/rxForm.js?0d03ed1d97ecc3d5da22e552e530565d150a8130:328:72)
	    at Context.<anonymous> (base/src/rxForm/rxForm.spec.js?eb992222af5bee3d1e04f8367e8fe4f0534c7b06:468:35)
Chrome 43.0.2357 (Mac OS X 10.10.3): Executed 635 of 635 (4 FAILED) (5.738 secs / 0.773 secs)
```

**Comment by CITguy** [comment]
TL;DR - Chrome 43 moved DOM Attributes to the prototype chain, so logic like `_.has()` no longer works.

See also https://youtu.be/sRtB6fVLf2w?t=1m18s
